core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -307,16 +307,44 @@ class HttpGoogleServicesDAO(val clientSecrets: GoogleClientSecrets,
     }
   }
 
-  override def setBucketLifecycle(bucketName: String, lifecycle: List[BucketInfo.LifecycleRule]): Future[Unit] =
+  override def setBucketLifecycle(bucketName: String,
+                                  lifecycle: List[BucketInfo.LifecycleRule],
+                                  userProject: GoogleProjectId
+  ): Future[Unit] =
     googleStorageService
-      .setBucketLifecycle(GcsBucketName(bucketName), lifecycle)
+      .setBucketLifecycle(
+        GcsBucketName(bucketName),
+        lifecycle,
+        bucketTargetOptions = List(BucketTargetOption.userProject(userProject.value))
```

Comment:
This passes the project so that when requester pays is on the workspace settings requests are accepted. I added the same line to all three settings in the DAO. I could break it out into a separate function but it seems straightforward enough to include multiple times. Alternate opinions are welcome.